drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -102,4 +102,29 @@ config NRF700X_WIFI_RX_MAX_DATA_SIZE
 	int "Maximum size of RX data"
 	default 1600
 
+config NRF700X_WIFI_HIGH_PERF_MODE
+	bool "Enable high performance mode"
+
+# Defaults optimized for Zperf, but can be overridden by user
+if NRF700X_WIFI_HIGH_PERF_MODE
```

Comment:
Do we need WIFI along with NRF700X? As in just call this NRF700X_HIGH_PERF_MODE?

Reply:
We do have Coex, so, added WIFI to disambiguate in case we comeup with WIFI_COEX_HIGH_PERF

Reply:
Thinking again, went with your suggestion.